### DISCUSSION TITLE
Make right click menu act on window under mouse cursor instead of act on current active window.

### DISCUSSION THREAD

**Discussion by gmcui** [f1multiviewer/issue-tracker#423] [Feature requests] [answered]
Currently, after moving mouse over to my target window, if I forget to left click first (which happens a lot), I right click and say change stream only to be surprised that a different window (which was active) changed.  It would be very nice if this can be fixed.

**Comment by LapsTimeOFF** [comment, marked as answer]
This has been resolved.
